scripts/deploy-pr.sh

```diff
@@ -71,8 +71,9 @@ fi
 gh_auth
 
 # get branch name and pr number
-branchName=$(gh pr view --json headRefName | jq -r .headRefName)
-prNumber=$(gh pr view --json number | jq -r .number)
+info=$(gh pr status --repo=coder/coder --json headRefName,number --jq '.createdBy[0]')
```

Comment:
This didn't support forks before, but you cannot specify `--repo` in `pr view` so I changed it to `pr status`.